### PR TITLE
add Cassandra host repo

### PR DESCRIPTION
**Pull request by BrandonArp**

### CODE REVIEW

**Comment by BrandonArp** [comment]
@vjkoskela Please review

File: app/models/cassandra/Host.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        metricsSoftwareState = value;
+    }
+
+    /**
+     * Converts this model into an {@link models.internal.Host}.
+     *
+     * @return a new internal model
+     */
+    public models.internal.Host toInternal() {
+        final DefaultHost.Builder builder = new DefaultHost.Builder()
```
Comment (vjkoskela) [inline app/models/cassandra/Host.java]: You should convert to thread local builder. At least any new builders, and preferably back-fill in a separate PR.

File: app/models/ebean/Host.java [hunk 2]
```diff
@@ -133,5 +135,18 @@ public Organization getOrganization() {
...
         this.organization = organizationValue;
     }
+
+    /**
+     * Converts the EBean model to an internal model.
+     *
+     * @return an internal model
+     */
+    public models.internal.Host toInternal() {
+        return new DefaultHost.Builder()
```
Comment (vjkoskela) [inline app/models/ebean/Host.java]: Same comment about builder invocation.

File: test/resources/logback-test.xml [hunk 3]
```diff
@@ -36,6 +36,7 @@
   <logger name="global" level="DEBUG" />
   <logger name="models" level="DEBUG" />
   <logger name="org.reflections.Reflections" level="ERROR" />
+  <logger name="com.arpnetworking.metrics.impl.ApacheHttpSink" level="OFF" />
```
Comment (vjkoskela) [inline test/resources/logback-test.xml]: Hrm. Why?

**Pull request merged**